nabavka/src/main/java/raf/si/racunovodstvo/nabavka/controllers/KalkulacijaController.java

```diff
@@ -34,18 +35,43 @@ public KalkulacijaController(IKalkulacijaService kalkulacijaService) {
     }
 
     @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> searchKalkulacija(
+    public ResponseEntity<Page<Kalkulacija>> searchKalkulacija(
         @RequestParam(name = "search", required = false, defaultValue = "") String search,
         @RequestParam(defaultValue = ApiUtil.DEFAULT_PAGE) @Min(ApiUtil.MIN_PAGE) Integer page,
         @RequestParam(defaultValue = ApiUtil.DEFAULT_SIZE) @Min(ApiUtil.MIN_SIZE) @Max(ApiUtil.MAX_SIZE) Integer size,
         @RequestParam(defaultValue = "brojKalkulacije") String[] sort
     ) {
         if (search.isEmpty()) {
-            return ResponseEntity.ok(this.kalkulacijaService.findAll());
+            return ResponseEntity.ok(this.kalkulacijaService.findAll(any()));
         }
 
         Pageable pageSort = ApiUtil.resolveSortingAndPagination(page, size, sort);
```

Comment:
Da li ovo radi sa any()?
Meni se cini kao da bi bilo ok resenje da se if sa 44. linije spusti ispod 48. linije, i da se unutar njega pozove: 
`return ResponseEntity.ok(this.kalkulacijaService.findAll(pageSort));`